src/pixelator/pna/resources/panels/proxiome-immuno-155-v2.csv

```diff
@@ -0,0 +1,167 @@
+# ---
+# name: proxiome-immuno-155-v2
+# aliases:
+#   - proxiome-immuno-155plex-v2
```

Comment:
I think we should set this as the new default by renaming the old panel file to `proxiome-immuno-155-v1.csv` and then se the alias for this to `proxiome-immuno-155` so that that this becomes the new default panel.